lib/i18n/js.rb

```diff
@@ -219,6 +219,9 @@ def self.export_i18n_js
         FileUtils.mkdir_p(export_i18n_js_dir_path)
 
         i18n_js_path = File.expand_path('../../../app/assets/javascripts/i18n.js', __FILE__)
+        destination_path = File.expand_path("i18n.js", export_i18n_js_dir_path)
+        return if File.exist?(destination_path) && File.identical?(i18n_js_path, destination_path)
```

Comment:
Is this one supposed to compare file contents as well? As far as I can tell [from the docs](https://ruby-doc.org/core-2.2.3/File.html#method-c-identical-3F), `identical?` does not check the file contents, but only for links etc.

Reply:
[Which is fixed afterward](https://github.com/fnando/i18n-js/commit/a88b206644a14b0de7bbb58e0ba1baab93737a8e)